### ISSUE TITLE
let `self-upgrade` trigger `install-patchperl`

### ISSUE THREAD

**Issue by gugod** [gugod/App-perlbrew#651]
This article from atoomic:

http://blogs.perl.org/users/atoomic/2018/10/using-perlbrew-after-macos-mojave-1014-update.html

Mentioned:

<blockquote>
In order to fix it I had to update install-patchperl using perlbrew command line and force to override. (I was under the incorrect assumption that 'perlbrew self-upgrade' would take care of this for me :-)
</blockquote>

Perhaps it'll be more friendly to automacially run `install-patchperl` as part of `self-upgrade`.

**Comment by hakonhagland** [comment]
Good idea! See also: https://github.com/bingos/devel-patchperl/issues/23
